sdk/storage/Azure.Storage.Blobs/src/BlobContainerClient.cs

```diff
@@ -854,7 +854,7 @@ private void SetNameFieldsIfNull()
                 catch (RequestFailedException storageRequestFailedException)
                 when (storageRequestFailedException.ErrorCode == BlobErrorCode.ContainerAlreadyExists)
                 {
-                    response = default;
+                    response = Response.FromValue(new BlobContainerInfo(), default);
```

Comment:
When I full-text-search repo for `CreateIfNotExistsInternal` I can see more places like that. I.e. in BlobClients. Should we apply same fix across the board ?